src/kOS/Screen/KOSToolbarWindow.cs

```diff
@@ -112,6 +112,7 @@ public void Awake()
             GameEvents.onGUIApplicationLauncherUnreadifying.Add(RemoveButton);
```

Comment:
The changes in this file are only here because a different PR hasn't been merged into develop yet, and I forgot it was still in my local copy.  When that PR is merged, this diff should go away.